web/src/views/NewIdea.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="container-app  container-extra-padding">
```

Comment:
I just noticed this final thing. The extra padding for this container causes the page to exceed the viewport width thereby activating the vertical scrollbar. I think it'd be cleaner without the scrollbar.

Reply:
The last two commits should fix that